src/plugins/auto/auto_executable_network.cpp

```diff
@@ -175,11 +175,15 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
             }
         }
         return decltype(ov::available_devices)::value_type {exeDevices};
+    } else if (name == ov::model_name) {
+        if (_autoSchedule->_loadContext[ACTUALDEVICE].future.valid()) {
+            _autoSchedule->_loadContext[ACTUALDEVICE].future.wait();
+        }
```

Comment:
I think we don't need to wait for this future

Reply:
Updated.